src/styles/spacing/index.md.njk

```diff
@@ -77,35 +81,67 @@ The Design System uses a responsive spacing scale which adapts based on screen s
   </tbody>
 </table>
 
-### Spacing override classes
-
-Occasionally, you might need to make minor adjustments like adding or removing spacing to elements of your design. You can use the responsive spacing override classes for this.
-
-
-### Using the override classes
-
-The spacing override classes are structured to allow you to apply any size of the scale, using margin or padding in any direction.
-
-For example, `govuk-!-margin-9` will set a margin of 60px on large screens and 40px on small screens.
-
-To add padding use `-padding` instead of `-margin`.
+## Static spacing
 
-If you want to add the margin or padding in a particular direction, add `left` for left, `right` for right, `top` for top, or `bottom` for bottom. For example, `-margin-top` will set margin-top, `-padding-right` will set padding-right.
-If you do not specify a direction, the margin or padding will be applied to all sides of the element.
+The static spacing scale is the same as the spacing for 'large screens' in the responsive spacing scale, but always uses the same spacing regardless of the screen width.
```

Comment:
Might be better to flip this, and match the way 'Responsive' is explained?

```suggestion
The static spacing scale stays the same for all screen sizes.

It uses the same spacing as 'large screens' in the responsive spacing scale, except it does not change based on screen width.
```

Reply:
It feels like we're repeating ourselves a bit with 'stays the same for all screen sizes' and 'except it does not change based on screen width'

How about

```suggestion
The static spacing scale stays the same for all screen sizes, and uses the same spacing as 'large screens' in the responsive spacing scale.
```